minichain/p2p.py

```
@@ -306,22 +306,32 @@ async def _listen_to_peer(
-                disconnected.append((reader, writer))
-        for reader, writer in disconnected:
+                return (reader, writer)
+
+        results = await asyncio.gather(
+            *(_send(r, w) for r, w in peers_snapshot)
+        )
+
+        for pair in results:
+            if pair is None:
+                continue
+            reader, writer = pair
```

Ruff (0.15.14)

[warning] Unpacked variable `reader` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)